Documentation for New Webhooks Integration API (#7004)
DataDog/documentation: content/en/api/integrations_webhook/webhooks_update.md

---
title: Update a Webhook for Webhooks Integration
type: apicontent
order: 21.3
external_redirect: /api/#update-a-webhook-for-webhooks-integration
---

## Update a Webhook for Webhooks Integration

Updates the endpoint with the name `<WEBHOOK_NAME>`.

All arguments associated with this request are optional

**ARGUMENTS**:

> -   **`name`**: The name of the webhook. Corresponds with `<WEBHOOK_NAME>`. [Learn more on how to use it in monitor notifications][1].

Reviewer: ```suggestion
-   **`name`**: The name of the webhook. Corresponds with `<WEBHOOK_NAME>`. For more information on using it, see [monitor notifications][1] docs.
```

> -   **`url`**: URL of the webhook

Reviewer: ```suggestion
-   **`url`**: URL of the webhook.
```

> -   **`payload`**: If `null`, uses default payload. If given a JSON payload, the webhook will return the payload specified by the given payload. [Using those variables][2].

Reviewer: ```suggestion
-   **`payload`**: If `null`, uses default payload. If given a JSON payload, the webhook will returns the specified payload. [Using those variables][2].
```

Reviewer: see comment about giving more link context

> -   **`custom_headers`**: If `null`, uses no header. If given a JSON payload, these will be headers attached to your webhook.

Reviewer: ```suggestion
-   **`custom_headers`**: If `null`, uses no header. If given a JSON payload, attaches specified headers to your webhook.
```

> -   **`encode_as`**: Encoding type. Can be given either `json` or `form`.

Reviewer: ```suggestion
-   **`encode_as`**: Encoding type. Valid values are `json` and `form`.
```
